bigtable_rs/src/bigtable.rs

```diff
@@ -316,42 +315,58 @@ impl BigTableConnection {
 
                 let auth_manager = Some(Arc::new(authentication_manager));
                 Ok(Self {
-                    client: create_client(endpoints, auth_manager, is_read_only),
+                    client: create_client(endpoints, None, auth_manager, is_read_only),
                     table_prefix: Arc::new(table_prefix),
                     timeout: Arc::new(timeout),
                 })
             }
         }
     }
 
-    fn new_with_emulator(
+    pub fn new_with_emulator(
```

Comment:
Do this need to be public?

Reply:
I'm creating a new bigtable connection pointing to a new emulator once per test case.

I don't want to have the test case modify its own environment variables to steer to the right emulator (smells racy), I'd prefer to be able to explicitly control where the emulator connects to through code, not ambient environment variables.

Maybe we can move things around a bit as part of https://github.com/liufuyang/bigtable_rs/pull/72#discussion_r1545315034, and come up with a slightly more idiomatic public interface.

Reply:
I added a docstring to this function, which now mentions when this should be used.